src/linear_algebra/bilinear_form.lean

```diff
@@ -1453,4 +1453,72 @@ begin
   erw [add_right, show B m.1 y = 0, by rw hB₂; exact m.2 y hy, hm, add_zero]
 end
 
+section linear_adjoints
+
+lemma comp_left_apply_eq_unique (B₁ B₂ : bilin_form R₁ M₁) (hB₂ : B₂.nondegenerate)
+  {φ ψ : M₁ →ₗ[R₁] M₁} (hφ : ∀ v w, B₁ v w = B₂ (φ v) w) (hψ : ∀ v w, B₁ v w = B₂ (ψ v) w) :
```

Comment:
Does it make sense to state this as an equality? It certainly makes the statement shorter and more similar to the name:
```suggestion
  {φ ψ : M₁ →ₗ[R₁] M₁} (hφ : B₁ = B₂.comp_left φ) (hψ : B₁ = B₂.comp_left ψ) :
```
`linear_map.eq_of_bilin_form_comp_left_eq` _might_ be a clearer name, but it's a bit of a mouthful, so I mention it only to give you other ideas. `bilin_form.comp_left_ext` might be another good name.

Reply:
I don't think mouthfulness is a concern for mathlib, so I think `linear_map.eq_of_bilin_form_comp_left_eq` is good

Reply:
Actually, I would like to keep the `bilin_form` namespace since then, I can use the dot notation. So I think I prefer `bilin_form.comp_left_ext` for that reason.

Reply:
Done!